app/global_errors.js

```diff
@@ -10,6 +10,9 @@ let quitText = 'Quit';
 let copyErrorAndQuitText = 'Copy error and quit';
 
 function handleError(prefix, error) {
+  if (console._error) {
+    console._error(`${prefix}:`, Errors.toLogFormat(error));
+  }
```

Comment:
what is this about?

Reply:
No idea, in the commit that they had in signal it had this, so i just put it in

Reply:
ok did some quick research, looks like js/logger.js hijacks all the normal console.* functions and maps them to console._* and replaces the normal one with the logger hijacked versions. so using _error is to get around the logger.